docs/features/read-only.md

```diff
@@ -18,14 +18,48 @@ The feature may also be used to view content that should not be edited, like fin
 
 ## Demo
 
-Use the demo below to toggle between editing modes and test the feature.
+Use the demo below to toggle between editing modes and test the feature. Some features, like exports or search, are still functional even in the read-only mode. While the search is available, the replace function, however, is disabled, as changing the content is blocked.
 
 {@snippet features/read-only}
 
+<info-box>
+	You can see that after switching to read-only mode, some of the toolbar items are still active and functional. It happens thanks to the {@link module:core/command~Command#affectsData `affectsData` property}. For most of the plugins, it is set to `true` by default, which makes them inactive when entering read-only mode. However, for those plugins that do not make any changes in the model &ndash; do not affect the content &ndash; it is set to `false`, thus allowing to still make use of them in modes with restricted user write permissions.
+</info-box>
+
+## Hiding toolbar in read-only mode
+
+Some use cases might require hiding the editor toolbar when entering the read-only mode. This can be achieved easily with the following code:
+
+```js
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		// ...
+	} )
+	.then( editor => {
+		const toolbarContainer = editor.ui.view.stickyPanel;
+
+		editor.on( 'change:isReadOnly', ( evt, propertyName, isReadOnly ) => {
+			if ( isReadOnly ) {
+				editor.ui.view.top.remove( toolbarContainer );
+			} else {
+				editor.ui.view.top.add( toolbarContainer );
+			}
+		} );
+	} )
+	.catch( error => {
+		console.log( error );
+	} );
```

Comment:
I'm not sure about this approach. Basically, it refers to [StickyPanelView](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/api/module_ui_panel_sticky_stickypanelview-StickyPanelView.html), not [ToolbarView](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/api/module_ui_toolbar_toolbarview-ToolbarView.html) itself. As I can see, the sticky panel is used only in the classic editor, this code can't be used with other editor types that use the toolbar, e.g. the decoupled editor.

How about using CSS for hiding and showing the toolbar? Something like this:

```javascript
const toolbarElement = editor.ui.view.toolbar.element;

editor.on( 'change:isReadOnly', ( evt, propertyName, isReadOnly ) => {
    if ( isReadOnly ) {
        toolbarElement.style.display = 'none';
    } else {
        toolbarElement.style.display = 'flex';
    }
} );
```

The above code snippet will work both with the classic editor and the decoupled editor.